test/mri/tests/ruby/test_refinement.rb

```diff
@@ -1835,6 +1835,21 @@ def test_used_modules
     assert_equal [ref::RefB, ref::RefA], ref::Combined::USED_MODS
   end
 
+  def test_refinements
```

Comment:
Copied from: https://bugs.ruby-lang.org/projects/ruby-master/repository/git/revisions/54198c7b97d3d353f7ac233e0360034b6e7b6cb6/entry/test/ruby/test_refinement.rb#L1751

Reply:
Test comment for Slack notifications